### PR TITLE
Add zipcodestack.com

### PR DESCRIPTION
**Pull request by zipcodestack**
Completely free API for zip code validations.

Please ensure your submission ticks all of these boxes:

 - [ x ] This is Software as a Service not self hosted
 - [ x ] It has a free tier not just a free trial
 - [ x ] The submission mentions what is free
 - [ x ] The submission is not already present in the list
 - [ x ] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -332,6 +332,8 @@ Table of Contents
   * [numlookupapi.com](https://numlookupapi.com) - Free phone number validation API - 100k free requests / month.
   * [Volca](https://volca.io#api) - Free API providing lists of technologies such as programming languages and database systems. Unlimited free requests.
   * [Query.me](https://query.me) - Collaborative data notebooks that execute in a script-like fashion and allow to fetch and send data via SQL, API, and a growing number custom blocks, like Slack and Email. Free for small Teams.
+  * [Zipcodestack](https://zipcodestack.com) - Free Zip Code API, covering postal codes of all countries. 10000 free requests/month.
+
```
Comment (ripienaar) [inline README.md:336]: please remove the empty line.

Site has some issues and wont render in browsers due to https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Content-Type-Options

Comment (zipcodestack) [reply README.md:336]: Thank you, I removed the line; site issues are also fixed (broken deployment).

File: README.md [hunk 2]
```diff
@@ -332,6 +332,7 @@ Table of Contents
   * [numlookupapi.com](https://numlookupapi.com) - Free phone number validation API - 100k free requests / month.
   * [Volca](https://volca.io#api) - Free API providing lists of technologies such as programming languages and database systems. Unlimited free requests.
   * [Query.me](https://query.me) - Collaborative data notebooks that execute in a script-like fashion and allow to fetch and send data via SQL, API, and a growing number custom blocks, like Slack and Email. Free for small Teams.
+  * [Zipcodestack](https://zipcodestack.com) - Free Zip Code API, covering postal codes of all countries. 10000 free requests/month.
```
Comment (ripienaar) [inline README.md:335]: I go the landing page, enter a post code and do search...nothing happens?

Sorry this is not a site thats ready for prime time

Comment (zipcodestack) [reply README.md:335]: You are right - things were not quite ready, sorry about that. The search is now working as intended, the API has been running for a while.

Comment (ripienaar) [reply README.md:335]: none of these documentation links work.

Sorry, we're not your QA department, happy to let you know my hourly rate for this kind of work but expecting people to QA your product in a pull request is not appropriate.

We'll close this PR now.

Screenshot 2022-12-19 at 20 50 41

**Pull request closed without merging**